src/server/scripts/Commands/cs_misc.cpp

```diff
@@ -1325,16 +1325,10 @@ class misc_commandscript : public CommandScript
 
     static bool HandleMaxSkillCommand(ChatHandler* handler, char const* /*args*/)
     {
-        Player* SelectedPlayer = handler->getSelectedPlayer();
```

Comment:
Returns null if no session (?), and it will always return **target player** or **self**.
That's why this would be faster.